packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -767,6 +768,9 @@ export const uploadTemplateToS3 = async (
   amplifyMeta: $TSMeta,
 ): Promise<void> => {
   const cfnFile = path.parse(filePath).base;
+  if (context.input.options?.minify) {
```

Comment:
do we have at least one e2e test that's using this flag ? If not we should add one that hits this if and the other one in upload appsync files.

Reply:
Yeah, I'll confirm (and we could probably use some verification in general around 'is the file minified' in either case), and add some specific e2es if not.